packages/zstd/zstd.0.4/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+maintainer: "ygrek@autistici.org"
+homepage: "https://github.com/ygrek/ocaml-zstd"
+license: "BSD-3-Clause"
+authors: [ "ygrek" ]
+tags: [ "org:ygrek" "clib:zstd" ]
+doc: [ "https://ygrek.org/p/ocaml-zstd/api/index.html" ]
+dev-repo: "git+https://github.com/ygrek/ocaml-zstd.git"
+bug-reports: "https://github.com/ygrek/ocaml-zstd/issues"
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs
+    "@install"
+    "@doc" {with-doc}
+    "@runtest" {with-test}]
+]
+depends: [
+  "ocaml"
+  "dune" {>= "3.7"}
+  "ctypes" {>= "0.13.0"}
+  "integers"
+  ("extlib" {with-test} | "extlib-compat" {with-test})
+  "base-unix" {with-test}
+  "conf-zstd"
+]
+synopsis: "Bindings to zstd compression library"
```

Comment:
```suggestion
conflicts: [ "ocaml-option-bytecode-only "]
synopsis: "Bindings to zstd compression library"
```
due to 
```
#=== ERROR while compiling zstd.0.4 ===========================================#
# context              2.2.0~beta2~dev | linux/x86_32 | ocaml-base-compiler.5.1.1 | pinned(https://github.com/ygrek/ocaml-zstd/releases/download/v0.4/zstd-0.4.tbz)
# path                 ~/.opam/5.1/.opam-switch/build/zstd.0.4
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p zstd -j 31 @install
# exit-code            1
# env-file             ~/.opam/log/zstd-7-752706.env
# output-file          ~/.opam/log/zstd-7-752706.out
### output ###
# File "_build/default/stubs/libzstd__function_gen__Function_description__Functions", line 1, characters 0-0:
# Error: File unavailable: /home/opam/.opam/5.1/lib/ocaml/str/str.cmxa
# File "_build/default/stubs/libzstd__type_gen", line 1, characters 0-0:
# Error: No rule found for stubs/.zstd_stubs.objs/native/zstd_stubs.cmx
# File "_build/default/stubs/libzstd__type_gen", line 1, characters 0-0:
# Error: No rule found for stubs/.zstd_stubs.objs/native/zstd_stubs.o
# File "_build/default/stubs/libzstd__type_gen", line 1, characters 0-0:
# Error: No rule found for
# stubs/.zstd_stubs.objs/native/zstd_stubs__Libzstd__type_gen.cmx
# File "_build/default/stubs/libzstd__type_gen", line 1, characters 0-0:
# Error: No rule found for
# stubs/.zstd_stubs.objs/native/zstd_stubs__Libzstd__type_gen.o
# File "_build/default/stubs/libzstd__type_gen", line 1, characters 0-0:
# Error: No rule found for
# stubs/.zstd_stubs.objs/native/zstd_stubs__Type_description.cmx
# File "_build/default/stubs/libzstd__type_gen", line 1, characters 0-0:
# Error: No rule found for
# stubs/.zstd_stubs.objs/native/zstd_stubs__Type_description.o
```

Reply:
this doesn't look like bytecode error, it references cmx, bytecode-only compiler wouldn't know such files exist?
is linux/x86_32 bytecode only on ci?
i don't find build logs and i don't have a setup to test this but this change looks suspicious

Reply:
The 32 bit systems are all bytecode only in ocaml recent compilers